apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/environment/PortalLegacyMetaServerProvider.java

```diff
@@ -0,0 +1,97 @@
+package com.ctrip.framework.apollo.portal.environment;
+
+import com.ctrip.framework.apollo.core.utils.ResourceUtils;
+import com.ctrip.framework.apollo.portal.util.KeyValueUtils;
+import com.google.common.base.Strings;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * a fake class to replace com.ctrip.framework.apollo.core.internals.LegacyMetaServerProvider in apollo-portal
```

Comment:
This is not a fake class, it's just an implementation for apollo-portal only, so I think we could change the description a little bit

Reply:
Get it.